packages/flutter_ai_providers/README.md

```diff
@@ -115,6 +115,19 @@ final provider = OllamaProvider(
 );
 ```
 
+### Openwebui Provider
+
+- [Website](https://openwebui.com/)
+- [Supported models](https://ollama.com/search)
```

Comment:
Maybe we can link to https://openwebui.com/models instead?

Reply:
Done

Reply:
I think you didn't push this change